### PR TITLE
Update put person method to allow optional age and gender updates

### PR DESCRIPTION
**Pull request by brettminnie**

### CODE REVIEW

File: src/TransformCore/PHE/HayApi/PersistenceBundle/Tests/Controller/PersonControllerTest.php [hunk 1]
```diff
@@ -137,8 +137,7 @@ public function testPutPersonActionWithValidIdReturns200()
                 'title'     => 'Grand Overfiend',
                 'lastname'  => 'LAST NAME',
                 'firstname' => $person->getFirstName(),
-                'age'       => $person->getAge(),
-                'gender'    => $person->getGender(),
+                'age'       => $person->getAge() + 5,
```
Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Tests/Controller/PersonControllerTest.php]: Removed `gender`

Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Tests/Controller/PersonControllerTest.php]: Yeah, want to make sure if it's not passed it's not changed

Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Tests/Controller/PersonControllerTest.php]: Separate test case?

Comment (eddiejaoude) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Tests/Controller/PersonControllerTest.php]: Ah ok. Yeah maybe another test scenario?

Comment (brettminnie) [inline src/TransformCore/PHE/HayApi/PersistenceBundle/Tests/Controller/PersonControllerTest.php]: Will do, did think it may need it

**Pull request merged**